### ISSUE TITLE
Fix `invalid-attr` reports error for `:` in `class` attribute

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#564]
- Markuplint version: `2.11.0`
- Parser lang: HTML 
- Node.js version: `16.16.0`
- OS: Windows

## Describe the bug

First, Tailwind CSS uses `:` in `class` attribute.
https://tailwindcss.com/docs/responsive-design

And markuplint reports `:` as invalid value for `class` attribute.

```
error: 属性「class」の値の部分はCSS構文として「<ident-token>*」である必要があります (https://www.w3.org/TR/SVG/styling.html#ClassAttribute) (invalid-attr)
```

However, `class` attribute can include `:`, as my understanding.
(I read HTML Spec, but I couldn't find a sentence such as "`class` can't include `:`")
https://html.spec.whatwg.org/multipage/dom.html#classes

## Code Example

```html
<div class="md:flex">
...
</div>
```

## Steps To Reproduce

Lint HTML file including the `Code Example` with `markuplint:recommended` config

## The current behavior

markuplint reports error for `:` in `class`

## The expected behavior

markuplint doesn't report error for `:` in `class`

## Note

To avoid this error, I set `Any` for `class` attr.

```
  extends: ["markuplint:recommended"],
  rules: {
    "invalid-attr": {
      option: {
        attrs: {
          class: {
            type: "Any",
          },
        },
      },
    },
  },
```
